Contact folder

odufour7 opened the pull request:
- I removed names on the folium contact map 
- I changed the histogram to allow log scale
- I put a kde on linear scale (kernel density estimation)

Commit:
remove name subjects contact map

Commit:
contact histogram done

chraibi reviewed (changes requested):
Please do not upload binaries and your local txt files.

Commit:
contact

Commit:
animation+contacts+satellitemap

odufour7 committed:
Merge branch 'MADRAS-crowds:main' into main

Commit:
pull

odufour7 commented:
- I added a layer "satellite" in the map tab
- I added a tab animation "It's very slow"
- I added a graph in the contact tab and I modified the histogram

Commit:
fix plot cumulative contacts legend and save fig

Commit:
add *._* gitignore

Commit:
add *._* gitignore

Commit:
add *._*.csv gitignore

Commit:
rm ._*.csv

Commit:
add .pkl for my trajectories

Commit:
rm traj

Commit:
rm traj
